### ISSUE TITLE
Redefine a class will cause crash.

### ISSUE THREAD

**Issue by siuying** [kishikawakatsumi/JavaScriptBridge#19]
Try to re-define a class, such as following:

``` javascript
var MainViewController = JSB.defineClass('MainViewController : UITableViewController', {}, {});
var MainViewController = JSB.defineClass('MainViewController : UITableViewController', {}, {});
```

would crash the app.

It would be great if we can redefine a class dynamically.

**Comment by kishikawakatsumi** [comment]
Yes I know this issue.
I hava 2 plans to solve it.
First is just ignore after the 2nd time, second is overwrite previous definitions.

I haven't decided yet...

**Comment by siuying** [comment]
I think overwrite the previous definition is natural. After all if you redefine a JavaScript object it gets overwritten.

**Comment by siuying** [comment]
I was trying to write code on a REPL (https://github.com/siuying/IGJavaScriptConsole), if we can define the class in runtime, we can write apps without restart the simulator which is a huge plus.

**Comment by kishikawakatsumi** [comment]
Fix 09d44f34164866b35ffc0f491008591bcab37e85 4450f09a4b37b43eaf08d8970f18a29ed48dc18e 5a3a4192c919c631ec8af15186926bc5226b20c3 

I checked it working fine with some simple application and tests. 
https://github.com/kishikawakatsumi/JavaScriptBridge/blob/master/Tests/JavaScriptBridgeTests.m

But I'm not sure if this is correct.
Please try to use in various situations?
